local-dev-first-start-only.sh

```diff
@@ -410,6 +410,11 @@ echo "  - docker exec -u hydro-service hydroshare python manage.py migrate --fak
 echo
 docker $DOCKER_PARAM exec hydroshare python manage.py migrate --fake-initial --noinput
 
+echo
```

Comment:
This doesn't hurt anything but I don't think this is necessary in this local dev script.